dev/loadgen/configs/prod-benchmark-pvc.yaml

```diff
@@ -84,7 +84,7 @@ repos:
     cloneTarget: main
     score: 20
     workspaceImage: registry.hub.docker.com/gitpod/workspace-full:latest
-  - cloneURL: https://github.com/gitpod-io/template-python-flask
+  - cloneURL: https://github.com/gitpod-io/non-gitpodified-repo
```

Comment:
Is this related to performance? I ask because I thought that whether this repository is activated should be assured by integration tests and not relates in performance tests.

Reply:
It's not related to performance tests. I agree with your point 👍 .

From my point of view, the loadgen test should contain these workspaces
- stress testing
- regular workspace with prebuild
- regular workspace without prebuild, but with .gitpod.yml
- regular workspace without prebuild and without .gitpod.yml

Reply:
I'd certainly prefer a repository that makes sense.